crypto/crypto_aes.c

```diff
@@ -89,30 +91,32 @@ useaesni(void)
 #endif /* CPUSUPPORT_X86_AESNI */
 
 /**
- * crypto_aes_key_expand(key, len):
- * Expand the ${len}-byte AES key ${key} into a structure which can be passed
- * to crypto_aes_encrypt_block.  The length must be 16 or 32.
+ * crypto_aes_key_expand(key, keylen):
```

Comment:
I'm not sure I see the point of the global `s/len/keylen/` here.  Also, if you do have a reason for changing this, you should change the _aesni version too.